src/patchelf.cc

```diff
@@ -41,6 +41,7 @@
 #include <unistd.h>
 
 #include "elf.h"
+#include "patchelf.h"
```

Comment:
Not quite sure why patchelf.h is not found here.

Reply:
Fixed it! It was missing from the **automake** dist file.